foyer/tests/implemented_opls_tests.txt

```diff
@@ -1,55 +1,90 @@
+1-bromobutane
```

Comment:
How are all of these molecules added if the corresponding top and gro files aren't added anywhere in this PR? Shouldn't they be tested for correct parameterization [here](https://github.com/mosdef-hub/foyer/blob/d841959fbef94cdfe009bf3e8dfd9c15f6aec43d/foyer/tests/test_opls.py#L51):

Reply:
There are several molecules in the `opls_validation` directory that have the gro and top files, but they aren't included in the `implemented_opls_tests.txt` file, probably from having missing parameters. Most of the molecules added to testing already existed in `opls_validation`

Reply:
If you run `test_opls.py` on this PR you'll see it's running ~150 tests. If you run it on the main branch it is closer to 100 tests